cristin-biobank/src/main/src/java/no/unit/nva/model/nva/Biobanks.java

```diff
@@ -0,0 +1,48 @@
+package no.unit.nva.model.nva;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
+import static no.unit.nva.cristin.model.JsonPropertyNames.CONTEXT;
+import static no.unit.nva.cristin.model.JsonPropertyNames.ID;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import java.net.URI;
+import java.util.Collections;
+import no.unit.nva.cristin.biobank.model.nva.Biobank;
+import java.util.List;
+
+
+@JsonInclude(NON_NULL)
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+public final class Biobanks {
+    private static final String BIOBANKS_STRING = "biobanks";
+
+    @JsonProperty(CONTEXT)
+    private final URI context;
+    @JsonProperty(ID)
+    private final URI id;
+    @JsonProperty(BIOBANKS_STRING)
+    private final List<Biobank> biobankList;
+
+    @JsonCreator
+    public Biobanks(@JsonProperty(CONTEXT) URI context,
+                          @JsonProperty(ID) URI id,
+                          @JsonProperty(BIOBANKS_STRING) List<Biobank> biobanks) {
+        this.context = context;
+        this.id = id;
+        this.biobankList = Collections.unmodifiableList(biobanks);
```

Comment:
When looking at the json representation, biobanks looks better than biobankList. Put if you use biobankList, be consistent and give the constructor parameter the same name.